validator/accounts/accounts_list.go

```diff
@@ -221,3 +233,26 @@ func listRemoteKeymanagerAccounts(
 	}
 	return nil
 }
+
+func listValidatorIndices(ctx context.Context, km keymanager.IKeymanager, client ethpb.BeaconNodeValidatorClient) error {
+	pubKeys, err := km.FetchValidatingPublicKeys(ctx)
+	if err != nil {
+		return errors.Wrap(err, "could not get validating public keys")
+	}
+	var pks [][]byte
+	for i := range pubKeys {
+		pks = append(pks, pubKeys[i][:])
+	}
+	req := &ethpb.MultipleValidatorStatusRequest{PublicKeys: pks}
+	resp, err := client.MultipleValidatorStatus(ctx, req)
+	if err != nil {
+		return errors.Wrap(err, "could not request validator indices")
+	}
+	fmt.Println(au.BrightGreen("Validator indices:").Bold())
+	for _, idx := range resp.Indices {
+		if idx != math.MaxUint64 {
+			fmt.Println(idx)
```

Comment:
Can we print them alongside the first 8 characters of each public key for easier identification?

Reply:
Sure, added public keys to the command output.